src/pages/FrontPage.js

```diff
@@ -64,12 +65,14 @@ FrontPage.propTypes = {
   courses: PropTypes.object,
   externalCourses: PropTypes.object,
   isStudentMode: PropTypes.bool,
-  setModeTeacher: PropTypes.func
+  setModeTeacher: PropTypes.func,
+  t: PropTypes.func
 };
 
 function mapStateToProps(state) {
   return {
-    isStudentMode: state.isStudentMode
+    isStudentMode: state.isStudentMode,
+    t: getTranslator(state)
```

Comment:
Du glemte å oversette "Kom i gang!" og "Lærer/Veileder" (de to store knappene på forsiden).